MFVNeutralino/python/Vertexer_cfi.py

```diff
@@ -26,9 +28,11 @@
                                no_track_cuts = cms.bool(False),
                                min_seed_jet_pt = cms.double(30),
                                min_track_pt = cms.double(1),
+                               min_track_pt_loose = cms.double(0.9),
```

Comment:
Is this used anywhere, or can we drop it?

Reply:
Again, it was from Ang's study. Yes, it can be dropped.

Reply:
Dropping it now, thanks for confirming!